### PR TITLE
Update appv-release-notes-for-appv-for-windows.md

### PR DESCRIPTION
**Pull request by GrantMeStrength**
Updating information re: VS support, as per Paul Chapman's request.

### CODE REVIEW

**Comment by illfated** [comment]
I am trying to come up with a suggestion for a slight rewrite of the sentence, because the current wording sounds a bit like a teacher teaching students something (slightly overbearing, but that is probably me reading too much into it). I have thought of ways to use 'Microsoft' instead of "we", as well as using "Currently" instead of "Today".

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@lomayor Please review the changes introduces by an internal MS member as the time permits. Thanks.

File: windows/application-management/app-v/appv-release-notes-for-appv-for-windows.md [hunk 1]
```diff
@@ -145,6 +145,8 @@ App-V doesn't support Visual Studio 2012.
 
 **Workaround**: Use a newer version of Microsoft Visual Studio.
 
+Today, we do not support app virtualization with Visual Studio, whether using Microsoft’s App-V or third party solutions such as VMWare’s ThinApp. While it is possible that customers might find that Visual Studio works well enough for their purposes when running within one of these environments, at this time we are unable to address any bugs or issues found when running in a virtualized environment.
```
Comment (lomayor) [inline windows/application-management/app-v/appv-release-notes-for-appv-for-windows.md]: Please consider rewrite below. Also, we need to be specific about the Visual Studio versions we are describing here.

NOTE: Visual Studio 2012 currently doesn't support app virtualization, whether using Microsoft’s App-V or third party solutions such as VMWare’s ThinApp. While it is possible that Visual Studio can work well enough in virtualized environments, the Visual Studio team can't directly address problems encountered while running in a these environments.

**Comment by lomayor** [comment]
@e0i , @illfated, please see comment in the changes. Was this copy edited?

**Comment by illfated** [comment]
@lomayor : This PR content has not been copy edited, so we could use a review by @nenonix & @JohanFreelancer9 given that they are available. I looked briefly into it, but was unable to come up with a better way to reword the text or make it read more naturally for native English speakers.

File: windows/application-management/app-v/appv-release-notes-for-appv-for-windows.md (hunk 1 above)
Comment (JohanFreelancer9) [inline windows/application-management/app-v/appv-release-notes-for-appv-for-windows.md]:
```suggestion
Currently, Visual Studio 2012 doesn't support app virtualization, whether using Microsoft App-V or third party solutions such as VMWare ThinApp. While it is possible you might find that Visual Studio works well enough for your purposes when running within one of these environments, we are unable to address any bugs or issues found when running in a virtualized environment at this time.
```

Comment (lomayor) [reply windows/application-management/app-v/appv-release-notes-for-appv-for-windows.md]: Looks good. Please commit.

Comment (Dansimp) [reply windows/application-management/app-v/appv-release-notes-for-appv-for-windows.md]: @GrantMeStrength can you please commit the suggested rewrite? After that we can approve this PR. thx

**Comment by e0i** [comment]
@GrantMeStrength Please follow up with copy editing suggestions so that the PR can proceed with the review. Thanks.

**Comment by e0i** [comment]
@GrantMeStrength Any updates with the copy editing suggestions? Thanks.

**Comment by e0i** [comment]
@GrantMeStrength Following up for updates. Thanks.

**Comment by Dansimp** [comment]
added copyedits directly to branch. merging.

**Pull request merged**